src/components-examples/aria/grid/grid-table/grid-table-example.html

```diff
@@ -0,0 +1,100 @@
+<table ngGrid class="example-table">
+  <thead>
+    <tr ngGridRow class="example-header-row">
+      <th ngGridCell class="example-cell example-cell-checkbox">
+        <mat-checkbox
+          ngGridCellWidget
+          [focusTarget]="cb._inputElement"
+          [checked]="allSelected()"
+          [indeterminate]="partiallySelected()"
+          (change)="updateSelection($event.checked)"
+          aria-label="all rows"
+          #cb="matCheckbox"
+        ></mat-checkbox>
+      </th>
+      <th ngGridCell class="example-cell">Summary</th>
+      <th ngGridCell class="example-cell">Assignee</th>
+      <th ngGridCell class="example-cell">Tags</th>
+    </tr>
+  </thead>
+  <tbody>
+    @for (task of tasks(); track task) {
+      <tr ngGridRow class="example-row">
+        <td ngGridCell class="example-cell example-cell-checkbox">
+          <mat-checkbox
+            ngGridCellWidget
+            [focusTarget]="cb._inputElement"
+            [(ngModel)]="task.selected"
+            aria-label="row {{$index}}"
+            #cb="matCheckbox"
+          ></mat-checkbox>
+        </td>
+        <td ngGridCell class="example-cell example-cell-summary">
+          <div
+            role="button"
+            ngGridCellWidget
+            widgetType="editable"
+            (onActivate)="startEdit($event, task, summaryInput)"
+            (onDeactivate)="completeEdit($event, task)"
+            (click)="widget.activate()"
+            #widget="ngGridCellWidget"
+          >
```

Comment:
What is the reason for having the user call `widget.activate()` on click instead of this being handled internally?